test/cctest/test_url.cc

```diff
@@ -0,0 +1,69 @@
+#include "node_url.h"
+#include "node_i18n.h"
+
+#include "gtest/gtest.h"
+
+using node::url::URL;
+
+class URLTest : public ::testing::Test {
+ protected:
+  void SetUp() override {
+#if defined(NODE_HAVE_I18N_SUPPORT)
+    std::string icu_data_dir;
+    node::i18n::InitializeICUDirectory(icu_data_dir);
+#endif
+  }
+
+  void TearDown() override {}
+};
+
+TEST_F(URLTest, Simple) {
+  URL simple("https://example.org:81/a/b/c?query#fragment");
+
+  CHECK_EQ(simple.protocol(), "https:");
```

Comment:
Good point. I'd actually forgotten that `EXPECT_EQ` existed :-) I'll switch it up to use that instead.